src/writeData/components/PluginCreateConfigurationFooter.tsx

```diff
@@ -1,36 +1,54 @@
 // Libraries
-import React, {FC} from 'react'
+import React, {FC, useEffect, useMemo} from 'react'
 import {connect, ConnectedProps} from 'react-redux'
 
 // Components
 import {Button, ComponentColor, Overlay} from '@influxdata/clockface'
 
 // Actions
-import {
-  addPluginBundleWithPlugins,
-  createOrUpdateTelegrafConfigAsync,
-} from 'src/dataLoaders/actions/dataLoaders'
+import {createOrUpdateTelegrafConfigAsync} from 'src/dataLoaders/actions/dataLoaders'
+import {updateTelegraf} from 'src/telegrafs/actions/thunks'
 
 // Types
-import {BundleName} from 'src/types/dataLoaders'
+import {AppState, ResourceType, Telegraf} from 'src/types'
 import {PluginCreateConfigurationStepProps} from 'src/writeData/components/PluginCreateConfigurationWizard'
 
+// Selectors
+import {getDataLoaders} from 'src/dataLoaders/selectors'
+import {getAll} from 'src/resources/selectors'
+
 type ReduxProps = ConnectedProps<typeof connector>
 type Props = PluginCreateConfigurationStepProps & ReduxProps
 
 const PluginCreateConfigurationFooterComponent: FC<Props> = props => {
   const {
     currentStepIndex,
-    onAddPluginBundle,
     onDecrementCurrentStepIndex,
     onExit,
     onIncrementCurrentStepIndex,
     onSaveTelegrafConfig,
+    onUpdateTelegraf,
     substepIndex,
+    telegrafConfig,
+    pluginConfig,
   } = props
 
+  const shouldTelegrafUpdate = useMemo(() => {
+    if (telegrafConfig) {
+      return true
+    }
+    return false
```

Comment:
stylistic, not a big deal, but this can be compacted into:

```ts
return Boolean(telegrafConfig)
```

or the less explicit, less understandable-if-you're-not-a-js-developer:

```ts
return !!telegrafConfig
```

Reply:
Updated.